docs/examples/freesolv-imp-exp.rst

```diff
@@ -0,0 +1,67 @@
+.. _freesolv-imp-exp:
+
+Hydration Free Energy of part of the FreeSolv Database in Implicit and Explicit Solvent
```

Comment:
Maybe "Calculation of absolute hydration free energies from the FreeSolv Database"?